docs/authentication/saml/okta.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Okta Workforce
+description: Learn how to integrate Okta Workforce with Clerk using SAML SSO.
+---
+
+# Okta Workforce
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+    },
+    {
+      title: "Add the appropriate Clerk SDK to your application",
+      link: "/docs/quickstarts/overview",
+    },
+  ]}
+>
+
+- Use Okta Workforce to enable single sign-on (SSO) via SAML for your Clerk application.
+
+</TutorialHero>
+
+Integrating Okta Workforce with Clerk allows you to use it as an Identity Provider (IdP) for your Clerk application to authenticate your users and manage their access to your application via SAML SSO.
```

Comment:
```suggestion
You can use Okta Workforce as a SAML SSO Identity Provider (IdP) for user authentication in your Clerk apps.
```
Just trying to compress this sentence a bit, as it's a lot to take in

Reply:
We basically have that in the "You will learn" section in the tutorial hero, so what do you think about removing the intro paragraph altogether?

Reply:
That works too!